eval/open_llm_leaderboard/hellaswag_utils.py

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
can we pls remove it from this files, its mostly coming from eval harness.

Reply:
Of course, the harder part will be to tell Open Source Bot that this is correct :-D

Reply:
Actually not that hard...the file should be excluded by Open Source Bot now